src/g3log/loglevels.hpp

```diff
@@ -70,12 +70,12 @@ namespace g3 {
 }
 
 #if (defined(CHANGE_G3LOG_DEBUG_TO_DBUG))
-const LEVELS DBUG {g3::kDebugVaulue, {"DEBUG"}},
+const LEVELS DBUG {g3::kDebugVaulue, "DEBUG"},
```

Comment:
My suggestion is to keep the braces and instead changing the constructor to take a std::string.
Ref the discussion regarding this issue